src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/config/ObjectMapperConfiguration.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.config;
+
+import com.fasterxml.jackson.databind.BeanDescription;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationConfig;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.databind.ser.BeanPropertyWriter;
+import com.fasterxml.jackson.databind.ser.BeanSerializerModifier;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.FeatureToggleService;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Configuration
+@RequiredArgsConstructor
+public class ObjectMapperConfiguration {
+
+    private final FeatureToggleService featureToggleService;
+
+    @Bean
+    public ObjectMapper objectMapper(Jackson2ObjectMapperBuilder builder) {
```

Comment:
If I understand this right, the idea is that you don't want fields with null values to appear on the JSON version of the object.
Apparently, we can set up the ObjectMapper to have this behaviour as default.
https://stackoverflow.com/questions/19383720/is-there-a-way-to-jsonignore-by-default-and-a-tag-to-allow-serialization/19384076

Reply:
No, I don't think it's just filtering out null values - you might have case data with a value in some field which you want to erase - this way you actually want to send a null value to CCD.

Reply:
Why would you have data in this field if it's feature toggled?

Reply:
Even if it's null if will be serialised with null value and if CCD doesn't have this field in its config it will throw an error.